src/Microsoft.Identity.Web.OWIN/AppBuilderExtension.cs

```diff
@@ -238,5 +245,27 @@ public static IAppBuilder AddMicrosoftIdentityWebApp(
 
             return app.UseOpenIdConnectAuthentication(options);
         }
+
+        /// <summary>
+        /// Rejects the internal claims, if present.
```

Comment:
```suggestion
        /// The SDK injects 2 claims named uuid and utid into the ClaimsPrincipal, from ESTS's client_info. These represent 
        /// the home oid and home tid and together they form the AccountId, which MSAL uses as cache key for web site 
        /// scenarios. In case the app owner defines claims with the same name to be added to the ID Token, this creates 
        /// a conflict with the reserved claims Id.Web injects, and it is better to throw a meaningful error. See issue 2968 for details.
        
```